float64   time_start
float64   time_end
float64[] segment_times
int8      poly_order
float64[] coeffs_x
float64[] coeffs_y
float64[] coeffs_z
